cfgov/jinja2/v1/_includes/macros/video-player.html

```diff
@@ -55,7 +55,8 @@
     {% set button_pos   = options.button_pos or 'center' %}
     {% set is_flexible  = ( options.video.width or options.video.height ) is not defined %}
     <div class="video-player
-                video-player__{{ video_player }}"
+                video-player__{{ video_player }}
+                {{ 'featured-content-module_visual' if is_flexible == false }}"
```

Comment:
Instead of adding `featured-content-module_visual` how about just using `additional_classes` and passing in `featured-content-module_visual`?